com.unity.render-pipelines.high-definition/Documentation~/lens-flare-data-driven-asset.md

```diff
@@ -134,6 +136,7 @@ The Lens Flare Element asset has the following properties:
 | **Property**    | **Description**                                              |
 | --------------- | ------------------------------------------------------------ |
 | Colors | The range of colors that this asset applies to the lens flares. |
+| Rotation | Specify an angle (in degree) which will be used for each element incrementaly. |
```

Comment:
As above: 
| Rotation | The angle of rotation (in degrees) applied to each element incrementally. |